tests/Extensions/Hosting.UnitTests/HostBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Hosting.Internal;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.TimedScopes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Omex.Extensions.Hosting.UnitTests
+{
+	[TestClass]
+	public class HostBuilderExtensionsTests
+	{
+		[DataTestMethod]
+		[DataRow(typeof(ILogger<HostBuilderExtensionsTests>))]
+		[DataRow(typeof(ITimedScopeProvider))]
+		public void AddOmexServices_TypesRegistred(Type type)
```

Comment:
Typo in registered. Looks like this is a few places in the changelist

Reply:
I will resolve it in the follow-up pr https://github.com/microsoft/Omex/pull/168